crates/number/src/serialization.rs

```diff
@@ -66,6 +73,112 @@ where
     deserializer.deserialize_str(Visitor {})
 }
 
+/// Serialize and deserialize [`U256`] as a decimal string.
```

Comment:
I think the plan was to always serialize to a decimal string but still be able to deserialize from a decimal or hex string for backwards compatibility reasons.